src/toxcore/net_crypto/crypto_connection.rs

```diff
@@ -9,9 +9,8 @@ use toxcore::crypto_core::*;
 use toxcore::dht::packet::*;
 use toxcore::time::*;
 
-/// How often in seconds `CookieRequest` or `CryptoHandshake` packets should be
-/// sent
-pub const CRYPTO_SEND_PACKET_INTERVAL: u64 = 1;
+/// Interval in ms between sending cookie request/handshake packets.
+pub const CRYPTO_SEND_PACKET_INTERVAL: u64 = 1000;
```

Comment:
why?

Reply:
Because it's used as milliseconds more often: https://github.com/tox-rs/tox/pull/271/files#diff-dd71e0765ce00efdcbe43457539f6e18R565 https://github.com/tox-rs/tox/pull/271/files#diff-dd71e0765ce00efdcbe43457539f6e18R569

And I had a bug there when forgot to multiply it by 1000. So I decided that having it in ms is clearer.